Lib/concurrent/futures/process.py

```diff
@@ -211,7 +211,7 @@ def _sendback_result(result_queue, work_id, result=None, exception=None):
         result_queue.put(_ResultItem(work_id, exception=exc))
 
 
-def _process_worker(call_queue, result_queue, initializer, initargs):
+def _process_worker(call_queue, result_queue, initializer, initargs, maxtasksperchild=None):
```

Comment:
I suggest adding `*, ` before your argument to make it keyword-only. This applies to a couple of other places in your code where this argument appears.